src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -74,10 +74,16 @@ public function getProxyFactoryCode(Definition $definition, $id)
         $methodName = 'get'.Container::camelize($id).'Service';
         $proxyClass = $this->getProxyClassName($definition);
 
+        $generatedClass = $this->generateProxyClass($definition);
```

Comment:
just pass `$proxyClass` as optional argument to `generateProxyClass`. its a private method anyway.